README.md

```diff
@@ -140,16 +160,6 @@ That, of course, will send users to:
 
 (Or whatever Kubernetes release that docs branch is associated with.)
 
-## Branch structure
```

Comment:
The missing material I'd like to see in Branch Structure and staging is this: 

"When you create a pull request (either against master or the upcoming release), your changes are staged in a custom subdomain on Netlify so that you can see your changes in rendered form before the PR is merged. You can use this to verify that everything is correct before the PR gets merged. To view your changes:
 
- Scroll down to the PR's list of Automated Checks
 - Click "Show All Checks"
 - Look for "deploy/netlify"; you'll see "Deploy Preview Ready!" if staging was successful
 - Click "Details" to bring up the staged site and navigate to your changes"